src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpectatorView.cs

```diff
@@ -72,7 +83,11 @@ public class SpectatorView : MonoBehaviour
         /// </summary>
         [Tooltip("Default prefab for creating a mobile recording service visual.")]
         [SerializeField]
-        public GameObject defaultMobileRecordingServiceVisualPrefab = null;
+#if UNITY_IOS || UNITY_ANDROID
+        private GameObject defaultMobileRecordingServiceVisualPrefab = null;
+#else
+        private GameObject defaultMobileRecordingServiceVisualPrefab;
+#endif

```

Comment:
Why is this difference needed? Can't we always have the version with the " = null;" at the end?

Reply:
When you switch between ios/android and uwp it generates warnings based on whether there is a value declared thats unused or a value not declared. We could disable this warning which may look better than these if defs.

Reply:
i'll leave this this way for now unless we want to disable this specific warning.